dist/arch/skylobby

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+exec /usr/bin/java \
```

Comment:
@skynet-gh You will be able to change in the repo itself the binary to be used after this is merged and PKGBUILD is updated again in a new PR